plone/app/upgrade/v51/profiles.zcml

```diff
@@ -59,4 +59,13 @@
       provides="Products.GenericSetup.interfaces.EXTENSION"
       />
 
+  <genericsetup:registerProfile
+      name="to51rc1"
+      title="Upgrade profile for Plone 5.1beta5 to Plone 5.1rc1"
```

Comment:
Was there a beta5?

Reply:
after checking http://dist.plone.org/release/ it looks like there was no beta 5.
If so, then the PR raising the 5109 metadata version should be closed and the 5108/beta5 profile renamed in rc1 and my additions merged into the 5108 profile...
@esteele  ?

Reply:
I changed the folder name in 2afdb75171ecaeac7bf17cab6808727fe3d6bf6c to reflect that b5 was now rc1, but must have missed committing the changes to the profiles.zcml.